src/content/en/updates/2018/06/chrome-68-deps-rems.md

```diff
@@ -61,6 +55,18 @@ Group and support for it is now removed from Chrome.
 [Chromestatus Tracker](https://www.chromestatus.com/feature/5253912718213120) &#124;
 [Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=845288)
 
+## Deprecations
+
+Current deprecations are listed below. {% include "web/updates/_shared/deprecations.html" %}
```

Comment:
`{% include` should be on it's own line, with a blank line on either side.

Reply:
It works fine and these need to be a single paragraph.